cmd/thanos/sidecar.go

```diff
@@ -26,13 +28,14 @@ import (
 	"github.com/thanos-io/thanos/pkg/store"
 	"github.com/thanos-io/thanos/pkg/store/storepb"
 	"google.golang.org/grpc"
-	kingpin "gopkg.in/alecthomas/kingpin.v2"
+	"gopkg.in/alecthomas/kingpin.v2"
 )
 
 const waitForExternalLabelsTimeout = 10 * time.Minute
 
-func registerSidecar(m map[string]setupFunc, app *kingpin.Application, name string) {
-	cmd := app.Command(name, "sidecar for Prometheus server")
+func registerSidecar(m map[string]setupFunc, app *kingpin.Application) {
+	comp := component.Sidecar
```

Comment:
I would maybe inline those, would be more verbose.

Reply:
I inlined it in the `registerSidecar` function, not sure if you want to inline it everywhere even in the `runSidecar` ?